chart/templates/virtualmachine.yaml

```
@@ -0,0 +1,63 @@
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachine
+metadata:
+  name: {{ include "mixmode.fullname" . }}
+  labels:
+    {{- include "mixmode.labels" . | nindent 4 }}
+    kubevirt.io/vm: {{ include "mixmode.fullname" . }}
+spec:
+  running: true
+  template:
+    metadata:
+      labels:
+        kubevirt.io/vm: {{ include "mixmode.fullname" . }}
+        {{- include "mixmode.selectorLabels" . | nindent 8 }}
+    spec:
+      domain:
+        devices:
+          disks:
+            - disk:
+                bus: virtio
+              name: {{ include "mixmode.fullname" . }}
+            - name: cloudinitdisk
+              disk:
+                bus: virtio
+          interfaces:
+            - name: default
+              masquerade: {}
+              ports:
+                # Ports not known, including HTTP and SSH for now
+                - port: 22
+                - port: 80
+        resources:
+          requests:
```

```suggestion
          limits:
              cpu: 4 # from OVA, greater than 2 CPUs required due to`isolcpus` config.
```
Updated to be 4 vCPU to match the OVF's config.

Not really sure if this should be `requests:` or `limits:`. I switched to `limits:` to match https://kubevirt.io/user-guide/virtual_machines/dedicated_cpu_resources/#requesting-dedicated-cpu-resources